src/components/cards/menuCard/PEMenuCard.tsx

```diff
@@ -0,0 +1,89 @@
+import { type ReactElement } from 'react';
+import PECarusel from '~/components/standard/carusel/PECarusel';
+import { Icon } from '../../standard/icon/Icon';
+import PeopleEatIcon from '../../standard/icon/PEIcon';
+
+export interface PEMealCardProps {
+    onClick?: () => void;
+    title?: string;
+    chefName: string;
+    chefPicture?: string;
+    pricePerPerson?: number;
+    description?: string;
+    pictures?: string[];
+    categories?: string[];
+    kitchens?: string[];
+}
```

Comment:
No longer meal card props, but `PEMenuCardProps` and too many thing optional which should not be optional:

```suggestion
export interface PEMenuCardProps {
    onClick?: () => void;
    title: string;
    chefFirstNameName: string;
    chefProfilePictureUrl: string;
    pricePerPerson: number;
    description: string;
    imageUrls: string[];
    categories: string[];
    kitchens: string[];
}
```